lib/output/utils.js

```diff
@@ -12,6 +12,7 @@ function hasOwn(obj, prop) {
 const wrapperSymbol = Symbol("wrapper");
 const implSymbol = Symbol("impl");
 const sameObjectCaches = Symbol("SameObject caches");
+const ctorRegistrySymbol = Symbol.for("constructor registry");
```

Comment:
This should probably be something more like:
```suggestion
const ctorRegistrySymbol = Symbol.for("[webidl2js] constructor registry");
```

Since [`Symbol.for(…)`](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Symbol/for) returns a&nbsp;shared symbol, it’s&nbsp;better to&nbsp;choose a&nbsp;name that is&nbsp;less likely to&nbsp;accidentally conflict.

Reply:
Is there a particular reason to use Symbol.for() here instead of Symbol()?

Reply:
I choosed to use the `Symbol.for()` to have a single constructor registry per global object. The interfaces may originate from different packages (jsdom, whatwg-url, ...), and if we were to use `Symbol()` it would create on the same global object one constructor registry per package.

Reply:
Well, the&nbsp;symbol’s&nbsp;name should&nbsp;still be&nbsp;prefixed to&nbsp;avoid accidental&nbsp;conflicts with&nbsp;symbols from&nbsp;unrelated&nbsp;packages.

Reply:
Done.